openedx/features/partners/models.py

```diff
@@ -25,6 +26,16 @@ class Meta:
         verbose_name = "Partner"
         verbose_name_plural = "Partners"
 
+    def clean(self, *args, **kwargs):
+        user_limit = self.configuration.get("USER_LIMIT")
+        try:
+            int(user_limit)
+        except ValueError:
+            raise ValidationError({
+                'configuration': ValidationError('USER_LIMIT can only be an integer string or blank string'),
```

Comment:
this fails for the blank case as well. If we're doing this cant we use 

user_limit = self.configuration.get('USER_LIMIT, '')

if user_limit != '' and not user_limit.isdigit():
    raise ...

Reply:
double quotes btw

Reply:
tbh this will break as well if the user sets integer directly in user limit. 
so the first check would be 

if isinstance(user_limit, str):

Reply:
Thanks for catching this. I have made the necessary changes in the code